docs/sources/enterprise/access-control/provisioning.md

```diff
@@ -14,44 +14,144 @@ If you want to manage roles and built-in role assignments by API, refer to the [
+To delete a role, you can add a list of roles under the `deleteRoles` section in the configuration file. Such a deletion is performed after a role is inserted or updated.
+
+Here is an example YAML file to delete a role:
+```yaml
+# config file version
+apiVersion: 1
 
-### Create and remove built-in role assignments
+# list of roles that should be deleted
+deleteRoles:
+  - name: custom:reports:editor
+    orgId: 1
+    force: true
+```
 
-To create a built-in role assignment, you can add list of assignments under `builtInRoles` section in the configuration file, as an element of `roles`. To remove a built-in role assignment, leave `builtInRoles` list empty. 
+### Assign your custom role to specific built-in roles
 
-Note that it is only possibly to provision [organization local]({{< relref "./roles#built-in-role-assignments" >}}) assignments. For creating or updating _global_ assignments, refer to the [Fine-grained access control HTTP API]({{< relref "../../http_api/access_control.md" >}}).
+To assign roles to built-in roles, add said built-in roles to the `builtInRoles` section of your roles. To remove a specific assignment, remove it from the list.
+
+For example, the following role is assigned to an organization editor or an organization administrator:
+
+```yaml
+# config file version
+apiVersion: 1
+
+# Roles to insert/update in the database
+roles:
+  - name: custom:users:editor
+    description: "This role allows users to list/create/update other users in the organization"
+    version: 1
+    orgId: 1
+    permissions:
+      - action: "users:read"
+        scope: "users:*"
+      - action: "users:write"
+        scope: "users:*"
+      - action: "users:create"
+        scope: "users:*"
+    builtInRoles:
+      - name: "Editor"
+      - name: "Admin"
+```
 
 ## Manage default built-in role assignments
 
-During the startup, Grafana creates [default built-in role assignments]({{< relref "./roles#default-built-in-role-assignments" >}}) with [fixed roles]({{< relref "./roles#fixed-roles" >}}). You can remove and add back later those assignments by using provisioning.
+During startup, Grafana creates [default built-in role assignments]({{< relref "./roles#default-built-in-role-assignments" >}}) with [fixed roles]({{< relref "./roles#fixed-roles" >}}). You can remove and later restore those assignments with provisioning.
 
 ### Remove default assignment
 
-To remove default built-in role assignment, you can use `removeDefaultAssignments` element in the configuration file. You would need to provide built-in role name and fixed role name.
+To remove default built-in role assignments, use the `removeDefaultAssignments` element in the configuration file. You need to provide the built-in role name and fixed role name.
+
+Here is an example:
+```yaml
+# config file version
+apiVersion: 1
+
+# list of default built-in role assignments that should be removed
+removeDefaultAssignments:
+  - builtInRole: "Grafana server administrator"
```

Comment:
Built-In roles can only be "Viewer", "Editor",  "Admin", "Grafana Admin"
```suggestion
  - builtInRole: "Grafana Admin"
```

Reply:
OK, so in code, use those. In plain text, use Grafana server administrator, organization viewer, organization editor, or organization administrator.